### PR TITLE
HAAR-2061: Remove deprecated filed

### PR DESCRIPTION
**Pull request by ravmoj**
1 ) Removed staffId (userId has same info),
2) activeCaseLoadId : can be derived from 'me/caseloads' endpoint in 'nomis-user-roles-api'

### CODE REVIEW

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

File: server/data/manageUsersApiClient.ts [hunk 1]
```diff
@@ -9,8 +9,7 @@ export interface User {
   authSource?: string
   uuid?: string
   userId?: string
-  staffId?: number // deprecated, use userId
```
Comment (tpmcgowan) [inline server/data/manageUsersApiClient.ts:12]: Does this mean the wiremock stub should also be updated to remove `staffId` too? Here: https://github.com/ministryofjustice/hmpps-template-typescript/blob/92d8a3d19239a08686bb5de137bccdfe6b929c18/integration_tests/mockApis/manageUsersApi.ts#L15

Comment (ravmoj) [reply server/data/manageUsersApiClient.ts:12]: Thanks, removed from stub

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**